user/migrations/0010_auto_20180711_1230.py

```diff
@@ -1,18 +0,0 @@
-# Generated by Django 2.0.7 on 2018-07-11 12:30
-
```

Comment:
why are we deleting these migration files?

Reply:
Actually too many migration files are generated of user model due to constantly changing in user model, also I was getting errors for migrations thats why removed all migration files and ran migration which creates one migration file for user which holds all the information for user model.